src/coreclr/pal/inc/rt/intsafe.h

```diff
@@ -30,46 +30,6 @@
 #define HIDWORD(_qw)    ((ULONG)((_qw) >> 32))
 #define LODWORD(_qw)    ((ULONG)(_qw))
 
-#if defined(MIDL_PASS) || defined(RC_INVOKED) || defined(_M_CEE_PURE) \
```

Comment:
Can this file be deleted completely? The remaining definitions do not seem to be used or are duplicated in other files.

Reply:
It should be fine. `HIWORD` etc are declared in `pal_mstypes.h`, which is always included.